guides/source/action_cable_overview.md

```diff
@@ -165,6 +165,30 @@ end
 
 [`rescue_from`]: https://api.rubyonrails.org/classes/ActiveSupport/Rescuable/ClassMethods.html#method-i-rescue_from
 
+#### Command callbacks
+
+You can define callbacks to be execute before, after or around any channel interaction (subscribing, unsubscribing or performing actions). For example:
```

Comment:
```suggestion
You can define callbacks to be executed before, after or around any channel interactions
(subscribing, unsubscribing or performing actions). For example:
```

Reply:
Hey @palkan, I'm also wondering if we should define `command` somewhere in this guide (under Terminology?), as it isn't mentiond anywhere else in this guide.